x/params/module.go

```diff
@@ -195,10 +199,35 @@ func provideModule(in paramsInputs) paramsOutputs {
 		app.SetParamStore(k.Subspace(baseapp.Paramspace).WithKeyTable(types.ConsensusParamsKeyTable()))
 	}
 	m := runtime.WrapAppModule(NewAppModule(k))
+	govHandler := govv1beta1.HandlerRoute{RouteKey: proposal.RouterKey, Handler: NewParamChangeProposalHandler(k)}
 
-	return paramsOutputs{ParamsKeeper: k, BaseAppOption: baseappOpt, Module: m}
+	return paramsOutputs{ParamsKeeper: k, BaseAppOption: baseappOpt, Module: m, GovHandler: govHandler}
 }
 
-func provideSubSpace(key depinject.ModuleKey, k keeper.Keeper) types.Subspace {
-	return k.Subspace(key.Name())
+// TODO
+// optional paramtypes.KeyTable?
+type subSpaceInputs struct {
+	depinject.In
+
+	Key      depinject.ModuleKey
+	Keeper   keeper.Keeper
+	KeyTable *types.KeyTable `optional:"true"`
+}
+
+func provideKeyTable(k depinject.ModuleKey) *types.KeyTable {
+	// TODO there must be a better way?
+	if k.Name() != "gov" {
```

Comment:
@aaronc Do you see what I'm going for here and is there a better approach you can think of?

Reply:
A `KeyTable` should only be produced for "gov" or other modules if needed.  Maybe this is fine as-is if pulled up to a `params` configuration.